scalding-beam/src/test/scala/com/twitter/scalding/beam_backend/BeamBackendTests.scala

```diff
@@ -113,6 +115,18 @@ class BeamBackendTests extends FunSuite with BeforeAndAfter {
       )
   }
 
+  test("bufferedTake"){
+    beamMatchesSeq(
+      TypedPipe
+        .from(1 to 50)
+        .groupAll
+        .bufferedTake(100)
```

Comment:
just a note: this is going to be really bad in a real job without map-side aggregation: the key is `Unit` so there is only one key, so this would have each mapper send 100, then have the reducers pick 100 of those.

But with no mapside aggregation, all the data will be sent to the reducers, and they will throw away all but 100.

But we can add an issue and come back and address this.

Reply:
Opened a ticket for this.
https://github.com/twitter/scalding/issues/1952